blazor/tooltip/content.md

```diff
@@ -11,31 +11,115 @@ documentation: ug
 
-## Template content
+##  Displaying dynamic HTML Content
 
-Any text or image can be added to the Tooltip, by default. To customize the Tooltip layout or to create your own visualized element on the Tooltip, template can be used.
+Any type of formatted content can be displayed within the Tooltip by default. To enhance the Tooltip layout or create a custom visualized element, use the `<ContentTemplate>` tag to define the structure.
 
-Refer to the following code example to add formatted HTML content to the Tooltip.
+Refer to the following code example to add dynamic HTML content to the Tooltip. Use @(MarkupString) to render HTML content dynamically.
 
 ```cshtml
 @using Syncfusion.Blazor.Popups
 
-<SfTooltip ID="tooltip" IsSticky="true" Target="#target" Content="@Content">
-    <div id='container'>
-        <p>
-            A green home is a type of house designed to be
-            <a id="target">
-                <u>environmentally friendly</u>
-            </a> and sustainable. And also focuses on the efficient use of "energy, water, and building materials." As green homes
-            have become more prevalent we have also seen the emergence of green affordable housing.
-        </p>
-    </div>
+<SfTooltip ID="tooltip" Target="#target">
+    <ContentTemplate>
+        <div>
+            @((MarkupString)Content)
+        </div>
+    </ContentTemplate>
+    <ChildContent>
+        <div id='container'>
+            <p>
+                A green home is a type of house designed to be
+                <a id="target">
+                    <u>environmentally friendly</u>
+                </a> and sustainable. And also focuses on the efficient use of "energy, water, and building materials." As green homes
+                have become more prevalent we have also seen the emergence of green affordable housing.
+            </p>
+        </div>
+    </ChildContent>
 </SfTooltip>
-
 @code
 {
     string Content = "<div><b>Environmentally friendly</b> or environment-friendly, (also referred to as eco-friendly, nature-friendly, and green) are marketing and sustainability terms referring to goods and services, laws, guidelines and policies that inflict reduced, minimal, or no harm upon ecosystems or the environment.</div>";
 }
 ```
 
-![Blazor Tooltip with Content](images/blazor-tooltip-with-content.gif)
\ No newline at end of file
+![Blazor Tooltip with Content](images/blazor-tooltip-with-content.gif)
+
+
+## Template in Blazor Tooltip Component
+
+The Tooltip's Template property allows you to fully customize the layout or add any custom elements within the Tooltip. You can include any content or HTML elements as Tooltip content by defining them in the `<ContentTemplate>` tag of the Tooltip component.
```

Comment:
The Tooltip's Template property enables you to completely customize the layout and incorporate custom elements into the Tooltip. You can insert any content or HTML elements as Tooltip content by specifying them within the `<ContentTemplate>` tag of the Tooltip component.

Check out the following code example to see how to include an HTML template inside the Tooltip.

Reply:
Content changed as per suggested.